### PR TITLE
Improve breakout rooms reactivity

### PR DESCRIPTION
**Pull request by marcoambrosini**
### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
B | A

### 🚧 TODO

- [ ] ...

### 🏁 Checklist

- [ ] ⛑️ Tests (unit and/or integration) are included
- [ ] 📘 API documentation in `docs/` has been updated or is not required
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [ ] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue [hunk 1]
```diff
@@ -176,15 +176,23 @@ export default {
...
+		breakoutRoomsConfigured(newValue) {
+			if (newValue) {
+				this.getBreakoutRooms()
+			}
+		},
 	},
 
 	methods: {
+		getBreakoutRooms() {
+			if (this.breakoutRoomsConfigured) {
```
[the remark this answers is not in the document]
Comment (marcoambrosini) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:199]: the logic has changed a bit so marking as resolved

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue [hunk 2]
```diff
@@ -25,7 +25,7 @@
 		 breakout rooms -->
 		<div class="breakout-rooms__actions">
 			<div class="breakout-rooms__actions-group">
-				<template v-if="breakoutRoomsConfigured">
+				<template v-if="isInCall">
```
Comment (nickvergessen) [inline src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:28]: lets keep it always for now so you can stop after exiting the call

**Review by Antreesy — COMMENTED** [review]
Looks fine for me in terms of logic

**Review by nickvergessen — APPROVED** [review]
🐘

**Pull request merged**